File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/src/Com/Aspose/Words/Model/ModificationOperationResult.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Com.Aspose.Words.Model 
{
  /// <summary>
  /// result of the operation which modifies the original document and saves the result
  /// </summary>  
  public class ModificationOperationResult 
  {                       
        /// <summary>
        /// Link to the source document (source for the modification operation)
        /// </summary>
        /// <value>Link to the source document (source for the modification operation)</value>    
        public FileLink Source { get; set; }

        /// <summary>
        /// Link to the dest document (result of the modification operation)
        /// </summary>
        /// <value>Link to the dest document (result of the modification operation)</value>    
        public FileLink Dest { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class ModificationOperationResult {\n");
          sb.Append("  Source: ").Append(Source).Append("\n");
          sb.Append("  Dest: ").Append(Dest).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-Android/Aspose-Words-Cloud-SDK-for-Android/src/main/java/com/aspose/words/model/ModificationOperationResult.java
package com.aspose.words.model;

import com.aspose.words.model.Link;
public class ModificationOperationResult {
  private Link Source = null;
  private Link Dest = null;
  /**
	 * getSource
	 * Gets Link
	 * @return Source
	 */
  public Link getSource() {
    return Source;
  }

	/**
	 * setSource
	 * Sets Link
	 * @param Source Link
	 */
  public void setSource(Link Source) {
    this.Source = Source;
  }

  /**
	 * getDest
	 * Gets Link
	 * @return Dest
	 */
  public Link getDest() {
    return Dest;
  }

	/**
	 * setDest
	 * Sets Link
	 * @param Dest Link
	 */
  public void setDest(Link Dest) {
    this.Dest = Dest;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class ModificationOperationResult {\n");
    sb.append("  Source: ").append(Source).append("\n");
    sb.append("  Dest: ").append(Dest).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

